### DISCUSSION TITLE
Rule applied with wrong triggers - moved all transaction to wrong category

### DISCUSSION THREAD

**Discussion by e1kfL0ck** [firefly-iii/firefly-iii#8584] [Firefly III]
Hello, 

I'm using rules to move data in a category after CSV import. 

I have created a rule with wrong triggers, that moved all my transactions to a wrong category. 

I was able to move some of them back to the right categories by reapplying my other rules.

But some that were in "No Category" now are in the wrong category. How can I moved them back to the "No category" ?

Thanks.

---
## Edit

I found the option there is `Clear any category`.

**Comment by JC5** [comment]
You can apply a rule that has the action "remove the category"?
